chapter2/tests/t20/terraform/dev/ec2.tf

```diff
@@ -0,0 +1,37 @@
+module "ec2_instances" {
+  source        = "terraform-aws-modules/ec2-instance/aws"
+  count         = var.ec2_instances_count
+  name          = "${local.labels.ec2_instance}-${count.index + 1}"
+  ami           = var.ec2_ami_id
+  instance_type = var.ec2_instance_type
+  key_name               = aws_key_pair.wordpress_cluster.key_name
+  vpc_security_group_ids = [module.wordpress_ec2_sg.security_group_id]
+  iam_instance_profile   = "ssm-role"
+  subnet_id              = local.availability_zone_subnets[count.index % length(local.availability_zone_subnets)]
```

Comment:
use `element` vs `modulo` approach - which is always hard to read

```
➜ terraform console                     
> element(["a", "b", "c"], 1)
"b"
> element(["a", "b", "c"], 2)
"c"
> element(["a", "b", "c"], 3)
"a"
> element(["a", "b", "c"], 4)
"b"
> element(["a", "b", "c"], 5)
"c"
>  

```

Reply:
https://developer.hashicorp.com/terraform/language/functions/element

Reply:
> use `element` vs `modulo` approach - which is always hard to read
> 
> ```
> ➜ terraform console                     
> > element(["a", "b", "c"], 1)
> "b"
> > element(["a", "b", "c"], 2)
> "c"
> > element(["a", "b", "c"], 3)
> "a"
> > element(["a", "b", "c"], 4)
> "b"
> > element(["a", "b", "c"], 5)
> "c"
> >  
> ```

@dmitry-mightydevops
Done
 It was like that at first, I don't remember why I started looking for other solutions.